changelogs/fragments/304_pep632.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+- Replace distutils with included module in ansible-core 2.12 to address PEP 632 if available (https://github.com/ansible-collections/ansible.posix/issues/303).
```

Comment:
```suggestion
- Use vendored version of ``distutils.version`` instead of the deprecated Python standard library to address PEP 632 (https://github.com/ansible-collections/ansible.posix/issues/303).
```